docs/users/jsc.rst

```diff
@@ -3,64 +3,128 @@
+| imvapich2c     | 2016b-GDR                 | 5.4.0          | Intel 2016.4.258 | MVAPICH2 2.2-GDR       | 8.0.44 |                | GCCcore, iccifort             | Compiler+MPI+CUDA toolchain                              |
+| pmvapich2c     | 2016b-GDR                 | 5.4.0          | PGI 16.9         | MVAPICH2 2.2-GDR       | 8.0.44 |                | GCCcore, PGI                  | Compiler+MPI+CUDA toolchain                              |
+| gpsolf         | 2016b                     | 5.4.0          | GCC 5.4.0        | ParaStationMPI 5.1.5-1 |        | OLF 2016b      | GCCcore, GCC, gpsmpi          | Compiler+MPI+Math toolchain                              |
+| intel-para     | 2016b                     | 5.4.0          | Intel 2017.0.098 | ParaStationMPI 5.1.5-1 |        | MKL 2017.0.098 | GCCcore, iccifort, ipsmpi     | Compiler+MPI+Math toolchain                              |
+| intel          | 2016b                     | 5.4.0          | Intel 2017.0.098 | IntelMPI 5.1.3.181     |        | MKL 2017.0.098 | GCCcore, iccifort, iimpi      | Compiler+MPI+Math toolchain                              |
+| gmvolfc        | 2016b-GDR                 | 5.4.0          | GCC 5.4.0        | MVAPICH2 2.2-GDR       | 8.0.44 | OLF 2016b      | GCCcore, GCC, gmvapich2c      | Compiler+MPI+CUDA+Math toolchain                         |
+| imvmklc        | 2016b-GDR                 | 5.4.0          | Intel 2016.4.258 | MVAPICH2 2.2-GDR       | 8.0.44 | MKL 2017.0.098 | GCCcore, iccifort, imvapich2c | Compiler+MPI+CUDA+Math toolchain                         |
+| pmvmklc        | 2016b-GDR                 | 5.4.0          | PGI 16.9         | MVAPICH2 2.2-GDR       | 8.0.44 | MKL 2017.0.098 | GCCcore, PGI, pmvapich2c      | Compiler+MPI+CUDA+Math toolchain                         |
++----------------+---------------------------+----------------+------------------+------------------------+--------+----------------+-------------------------------+----------------------------------------------------------+
+
+*OLF 2016b: OpenBLAS 0.2.19, LAPACK 3.6.1, ScaLAPACK 2.0.2, FFTW 3.3.5
 
 Configuration
 ~~~~~~~~~~~~~
 
-*(to be completed)*
+EasyBuild is configured using a ``Developers`` module that sets up correctly all the EasyBuild related environment
+variables
+
+TODO: put the output of the command
+
+.. _users_use_cases_jsc_eb_notes:
 
 Notes
 ~~~~~
 
-*(none)*
+* JSC deploys software in "Stages". A stage is snapshot of the latest versions of all the available software at a given
+  time. The stages change every 6 months, with the arrival of new projects.
+* There is a development stage for testing software before moving it to production
+* The software is deployed in a shared GPFS filesystem
+* Software is deployed in production just by the infrastructure manager
+* The module cache is updated by cron job every 30 minutes
```

Comment:
missing `.`s ?

also, include link to http://hpcugent.github.io/easybuild/files/eb-jsc-hust16.pdf?